google-cloud-firestore/lib/google/cloud/firestore/query.rb

```diff
@@ -921,32 +921,33 @@ def self.start query, parent_path, client
           GREATER_THAN
           GREATER_THAN_OR_EQUAL
         ].freeze
-        ##
-        # @private
-        UNARY_NIL_VALUES = [nil, :null, :nil].freeze
-        ##
-        # @private
-        UNARY_NAN_VALUES = [:nan, Float::NAN].freeze
-        ##
-        # @private
-        UNARY_VALUES = (UNARY_NIL_VALUES + UNARY_NAN_VALUES).freeze
+
+        def value_nil? value
```

Comment:
Should these be `@private`?

Reply:
They are protected methods, so they are considered `@private` by default. Constants aren't ever `public`/`protected`/`private` and so that is why we mark them `@private`.

Reply:
Gotcha, couldn't see that in this PR view.